pkg/statsd/aggregator.go

```diff
@@ -187,8 +202,8 @@ func (a *MetricAggregator) Process(f ProcessFunc) {
 	f(a.metricMap)
 }
 
-func (a *MetricAggregator) isExpired(now, ts gostatsd.Nanotime) bool {
-	return a.expiryInterval != 0 && time.Duration(now-ts) > a.expiryInterval
+func (a *MetricAggregator) isExpired(interval time.Duration, now, ts gostatsd.Nanotime) bool {
```

Comment:
Can be a plain function rather than a method.

Reply:
Good catch, fixed